test/hcl_to_cai/tfplan_to_cai_test.go

```diff
@@ -1,18 +1,20 @@
-package conversion
+package caitest
```

Comment:
I am worried about circular dependency, since compute_instance.go is in a package that cai_to_tfplan/tfplan_to_cai references, since the test uses functions defined in cai_to_tfplan/tfplan_to_cai.

Reply:
yeah, makes sense. We can't do per-resource json -> HCL tests inside the hcl module because the converter is only responsible for asset -> terraform data, not the conversion of terraform -> HCL.

For the TF provider they basically get around this by having everything in one module but I'd rather avoid that if we can because it's hard to work with at scale.

I think that having per-resource integration tests is enough for now; it has clear inputs and outputs that map to user experience. If we want to have unit tests per converter we can add that later / separately.